install.sh

```diff
@@ -1027,6 +1027,7 @@ then
   execute_sudo "${MKDIR[@]}" /etc/paths.d
   echo "${HOMEBREW_PREFIX}/bin" | execute_sudo tee /etc/paths.d/homebrew
   execute_sudo "${CHOWN[@]}" root:wheel /etc/paths.d/homebrew
+  execute_sudo "${CHMOD[@]}" "a+r" /etc/paths.d/homebrew
```

Comment:
```suggestion
  execute_sudo "${CHMOD[@]}" 0644 /etc/paths.d/homebrew
```
So we're not at the mercy of the machine's `umask`?